instance/openstack_utils.py

```diff
@@ -265,5 +265,5 @@ def delete_swift_container(container_name, **kwargs):
     with swift_service(**kwargs) as service:
         # Service delete may yield arbitrary large amount of results,
         # and since it yields we should read all results.
-        for result in service.delete(container_name):  # pylint: disable=unused-variable
-            pass
+        results = [result for result in service.delete(container_name)]
+        del results[:]
```

Comment:
This is just a nitpick of mine since now we don't have to disable a pylint warning and also list comprehensions are faster than for loops. Since the whole point of this line is to consume the messages then it felt like a more efficient way to do it. Also, we can use `del` instead of letting the garbage collector free the memory used here.

Reply:
Yes, you are correct. It is not necessary to change it to a list comprehension.
Usually list comprehensions are faster than for loops when creating a list.
I also thought it would be easier to read by re-writing it like that since the whole point of the for loop is to just consume the response. There's nothing done with the response so it is explicitly deleted instead of waiting for the garbage collector to free up that memory. But by using a for loop and doing assigning the consumed response to a single variable the code doesn't pollute the memory and still the garbage collection will only have to free up the memory used by the last result in the list.
I changed it back to how it was.